internal/integration/cli/etcd.go

```diff
@@ -0,0 +1,35 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// +build integration_cli
+
+package cli
+
+import (
+	"github.com/talos-systems/talos/internal/integration/base"
+)
+
+// EtcdSuite verifies etcd command.
+type EtcdSuite struct {
+	base.CLISuite
+}
+
+// SuiteName ...
+func (suite *EtcdSuite) SuiteName() string {
+	return "cli.EtcdSuite"
+}
+
+// TestMembers etcd members should have some output.
+func (suite *EtcdSuite) TestMembers() {
+	suite.RunCLI([]string{"etcd", "members", "--nodes", suite.RandomDiscoveredNode()}) // default checks for stdout not empty
```

Comment:
I think we should limit that to control plane nodes only (`suite.RandomDiscoveredNode()` has an argument for that)